### ISSUE TITLE
templateExtractor fails when an empty string is passed to transloco pipe

### ISSUE THREAD

**Issue by ronnetzer** [ngneat/transloco-keys-manager#30]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

When an empty string is passed to `transloco` pipe, the cli crashes on find command with this error
```bash
TypeError: Cannot read property 'replace' of undefined
    at Object.forEachKey
    at forEach.e 
    at Array.forEach
    at templateExtractor
    at Object.extractKeys
    at Object.extractTemplateKeys
    at Object.buildKeys
    at Object.findMissingKeys
    at Object.<anonymous>
    at Module._compile
```

Also a side issue, the pipe regex in templateExtractor also look for keys in `translocoCurrency` pipe expressions. (in my project the empty string is passed to `translocoCurrency` pipe which caused the cli to crash)

## Expected behavior

Ignore empty strings passed to transloco pipe
Fix pipe regex to ignore translocoCurrency pipe

## Minimal reproduction of the problem with instructions

 Pass an empty string to `transloco` or `translocoCurrency` pipes

## Environment

<pre><code>
Angular version: 9.1.0
Keys manager version: 2.0.3

**Comment by shaharkazaz** [comment]
@ronnetzer Can you share the part of the templates that failed? just so I could test the fix on what currently fails 🙂

**Comment by ronnetzer** [comment]
@shaharkazaz Sure,
I was able to reproduce it on an empty component with nothing but
```HTML
<span>{{ 'test' | transloco }}</span>
<span>{{  '' | transloco }}</span>
```

comment the line with the empty string and it will work

**Comment by shaharkazaz** [comment]
@ronnetzer Thanks! I'll check it out 🙂

**Comment by shaharkazaz** [comment]
@ronnetzer fixed in v2.0.5 🙂
